spec/elastic_apm/spies/sneakers_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'sneakers'
+
```

Comment:
I think you'll have to require the spy here as well:

`require 'elastic_apm/spies/sneakers'`

Reply:
added